docs/source/introduction/function_summary.rst

```diff
@@ -1,8 +0,0 @@
-.. dropdown:: Distribution Statement
-
- | # # # This source code is protected under the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
-
-********************************
```

Comment:
Will this be automated with the CLI?

Reply:
Gwyn says this is tracked in her documentation timeline, so good with resolving.